composer.json

```diff
@@ -32,7 +32,8 @@
         "build:dev": "COMPOSER_VENDOR_DIR=vendor composer install",
         "build:prod": "COMPOSER_VENDOR_DIR=tawkto/vendor composer install --no-dev",
         "lint": "phpcs -p -s -v --runtime-set ignore_warnings_on_exit true .",
-        "lint:fix": "phpcbf -p -s -v .; err=$?; if [ $err -eq 1 ]; then exit 0; else exit $err; fi;"
+        "lint:fix": "phpcbf -p -s -v .; err=$?; if [ $err -eq 1 ]; then exit 0; else exit $err; fi;",
+        "package": "mkdir -p ./tmp/tawkto-live-chat && cp -r ./tawkto/* ./tmp/tawkto-live-chat && (cd ./tmp && zip -9 -rq ../tawkto-live-chat.zip ./tawkto-live-chat) && rm -r ./tmp"
```

Comment:
Would recommend putting the `.zip` in `tmp` alongside the folder archived so there's no need for 2 ignores.
Additionally - clean before packaging (can do `clean` script that does `rm -r ./tmp`)

Reply:
Fixed it in this commit.

https://github.com/alvinjohnsonso/tawk-wordpress/pull/31/commits/2222f698fa868aabdd6de99abe2ee2f4193ee35e